internal/notification/const.go

```diff
@@ -1,4 +1,13 @@
 package notification
 
+import (
+	"regexp"
+)
+
 const fileNotifierMode = 0600
 const rfc5322DateTimeLayout = "Mon, 2 Jan 2006 15:04:05 -0700"
+
+var (
+	// RegexpValidEmail matches if the email conforms to a known standard.
+	RegexpValidEmail = regexp.MustCompile(`^[0-9a-zA-Z+-_~]+@([a-zA-Z0-9]([a-zA-Z0-9-]+[a-zA-Z0-9])?\.){1,127}[a-z-A-Z]+$`)
```

Comment:
This regex will fail to catch valid 2 character domains like `x@me.com`.

I think it would probably make sense to replace the validation logic where this regex is used with: https://pkg.go.dev/net/mail#ParseAddress.